hack/create-operatorhub-prs.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+# Copyright 2023 Red Hat, Inc. and/or its affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+VERSION=$1
+if [[ $1 == v* ]]; then TAG=$1; else TAG=v$1; fi
+GITHUB_AUTHOR=$2
+COMMUNITY_OPERATORS=community-operators
+COMMUNITY_OPERATORS_PROD=community-operators-prod
+if [[ $3 == false ]]; then DRY_RUN=false; else DRY_RUN=true; fi
+if [ -z "$4" ]; then KOGITO_OPERATOR_DIR=$(pwd); else KOGITO_OPERATOR_DIR=$4; fi
+upstream_author=${UPSTREAM_AUTHOR:-kiegroup}
+
+if [ "$GITHUB_AUTHOR" = "$upstream_author" ]; then
+   echo "Upstream author and GitHub author are equal, no need to setup upstream${upstream_author}"
+else
+    echo "Upstream author and GitHub author are equal. Adding upstream repo"
+    git remote add upstream https://github.com/${upstream_author}/kogito-operator.git >/dev/null 2>&1
+fi
+
+echo "Kogito Operator directory is ${KOGITO_OPERATOR_DIR}"
+
+git fetch --all --tags
+echo "Checking out Kogito $TAG"
+git checkout tags/$TAG
+
+cd /tmp
+
+create_operatorhub_pr() {
+  echo "### Starting changes on $1 repo ####"
+  if [ -d "$1" ];
```

Comment:
it is better to always begin clean, I would suggest to do a rm -rf ...... 2&> /dev/null

clone and add the changes.

Reply:
If you would like to execute the script using an env where you already executed it with the dry_run there is no need to clone again the project.